demisto_sdk/tests/integration_tests/secrets_integration_test.py

```diff
@@ -84,32 +92,30 @@ def test_integration_secrets_integration_positive(mocker, tmp_path):
     Then
     - Ensure secrets validation succeed.
     """
-    integration_with_secrets_path = join(
-        DEMISTO_SDK_PATH, "tests/test_files/content_repo_example/Packs/FeedAzure/Integrations/FeedAzure/FeedAzure.yml"
-    )
+    # Mocking the git functionality (Else it'll raise an error)
+    mock_git(mocker)
+    # Change working dir to repo
+    pack = repo.create_pack('PackName')
+    integration = pack.create_integration('sample')
+    os.chdir(integration.repo_path)
+    secret_string = 'email@white.listed'
+    integration.write_yml({'this is a secrets': secret_string})
```

Comment:
write_yml will override the whole integration yml, and it will make it "invalid" yml.
Instead as we discussed, do one of the following:
a. `integration.set_description(secret_string)`
b. `integration.update_yml(description=secret_string)`
c. `integration = pack.create_integration(name='simple', description=secret_string)`